### ISSUE TITLE
Gather Lords' email addresses

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#322]
Just leaving this here as a 'nice to have'.

A recent campaign has caused a couple of disgruntled users who have seen the 'you've sent this message too many times' and 'you can't send identikit messages' errors to contact us.

This has highlighted the fact that on WTT, most Lords are contacted via the central HoL email address. Our abuse system kicks in when too many messages are sent to a single address, irrespective of whether that address covers several representatives.

Adding Lords' email addresses might circumvent this although I do note that the HoL itself discourages mass mailings. I suppose one good thing is that Lords generally hang around longer than other reps so the effort would have a longer benefit...

**Comment by MyfanwyNixon** [comment]
PS: Might be useful to know how many Lords need email addresses, then we could assess whether this needs an appeal to our users, or just half an hour from each member of staff.

**Comment by MyfanwyNixon** [comment]
Quite a few of them are on here: http://news.bbc.co.uk/democracylive/hi/representatives/index.stm
And if we wanted to take a punt, they appear to take the format SurnameInitial@parliament.uk.

**Comment by dracos** [comment]
No, they're definitely not all in that format. http://data.parliament.uk/membersdataplatform/services/mnis/members/query/House=Lords/Addresses/ has all the email addresses given on the Parliament website; we already scrape this as in e.g. https://github.com/mysociety/theyworkforyou/issues/2

**Comment by MyfanwyNixon** [comment]
Ah ok then.

**Comment by dracos** [comment]
Sorry, I mean we have that data, but we don't use it in WriteToThem currently. https://github.com/mysociety/internal-services/blob/master/services/mapit-dadem-loading/scrapers/lord is the script that moves Lord data from TWFY to WTT weekly, which would be where you'd probably want to do something like this (though the email data isn't stored there, fwiw).
